Instructions data download own machine 54 (#104)
ESMValGroup/ESMValTool_Tutorial: setup.md

If you are planning on running ESMValTool on your own machine, please make sure
that you are able to download CMIP data and that you have several GB of space
available to install conda & ESMValTool, but also enough to make a copy of some
data (12Gb).

> For this tutorial you can download the required data files by

Reviewer: ```suggestion
We can download a single data file following the instructions as described below:
```

> 1. Going to the [CMIP5 search page on the DKRZ ESGF node][cmip5-search]

Reviewer: ```suggestion
1. Go to the [CMIP5 search page on the DKRZ ESGF node][cmip5-search]
```

> 2. Performing the following search constraints

Reviewer: ```suggestion
2. Perform the following search constraints
```

    - Model = HadGEM2-ES
    - Experiment = historical
    - Time frequency = mon
    - Ensemble =  r1i1p1
    - Variable = thetaoga

3. Press search button

> 4. On single single search result press `List files`

Reviewer: ```suggestion
4. On single search result press `List files`
```

5. At bottom of page click on `Show All Files`
6. Find the file which starts with `thetaoga` in the list
7. Use `HTTP Download` link to download
   the [thetaoga_Omon_HadGEM2-ES_historical_r1i1p1_185912-200512.nc][theatoga.nc] file
   to `~/default_inputpath/` directory.

> The tutorial needs more data files. All of the data files should be downloaded

Reviewer: ```suggestion
However, the tutorial needs more data files. The [dataset.urls][ds] file contains all data set URLs and in comments the used search constraints. All of the data files should be downloaded
```

> to `~/default_inputpath/` directory with [wget](https://en.wikipedia.org/wiki/Wget):

Reviewer: ```suggestion
to `~/default_inputpath/` directory. 

To download the data, run the following command using [wget](https://en.wikipedia.org/wiki/Wget):
```

~~~shell
wget --no-clobber --input-file \
  https://github.com/ESMValGroup/ESMValTool_Tutorial/raw/master/data/dataset.urls \
  --directory-prefix $HOME/default_inputpath/
~~~

> The [dataset.urls][ds] file contains all data set urls and in comments the used search constraints.

Reviewer: ```suggestion
```
